_docker/drupal/drupal-filesystem/web/modules/custom/rhd_assemblies/src/Service/DownloadManagerApi.php

```diff
@@ -114,8 +114,8 @@ public function getContentById($id) {
           $request = $this->client->request('GET', $feed_url);
           $response = $request->getBody()->getContents();
           $data = json_decode($response);
-          // Persist this data to a download_manager_api cache bin for an hour.
-          $this->cacheBin->set($cid, $data, time() + 3600);
```

Comment:
Swap the 'hard coded' 3600 for the `$this->configFactory->get('redhat_developers')->get('downloadManager.ttl')` config variable

Reply:
Is it ever possible for $ttl to not have a value?  Probably unlikely, do we set fallback values when looking up configuration?

Reply:
Hmm. It wouldn't hurt to set a fallback. There is always a chance that a mistake was made setting the config variable in one of the `drupal/managed-platform/ansible/templates/[ENVIRONMENT]/drupal/rhd.settings.php.j2 ` files and/or that someone could accidentally remove one at some point in the future. I will make this change now. Thanks!